### PR TITLE
Adds IcingaDB-Web support

### PR DESCRIPTION
**Pull request by mocdaniel**
This PR adds IcingaDB-Web support to the Icingaweb2 role. It sets sane defaults and works on Debian and RHEL.

### CODE REVIEW

File: roles/icingaweb2/tasks/modules/icingadb_web.yml
Comment (mkayontour) [inline roles/icingaweb2/tasks/modules/icingadb_web.yml:1]: Please use the variable `_i2_config_hash` to reduce the amount of templates.

File: roles/icingaweb2/templates/icingadb_config.ini.j2
Comment (mkayontour) [inline roles/icingaweb2/templates/icingadb_config.ini.j2:1]: Should not be used, instead use `modules_config.ini.j2`

**Review by mkayontour — CHANGES_REQUESTED** [review]
Please change the commented sections and then it looks good to me. Thanks

File: roles/icingaweb2/tasks/modules/icingadb_web.yml
Comment (mocdaniel) [reply roles/icingaweb2/tasks/modules/icingadb_web.yml:1]: Done.

File: roles/icingaweb2/templates/icingadb_config.ini.j2
Comment (mocdaniel) [reply roles/icingaweb2/templates/icingadb_config.ini.j2:1]: Done.

**Comment by mocdaniel** [comment]
> Please change the commented sections and then it looks good to me. Thanks

I adjusted the role, but I needed to introduce a new file in order to loop over two tasks as a block sequentially.

This was needed because we have to set `_i2_config_hash` for each file to be generated.

**Review by mkayontour — CHANGES_REQUESTED** [review]
Can you change the include_tasks to the ansible fqcn name as mentioned in the comments ;).

File: roles/icingaweb2/tasks/modules/icingadb_web.yml [hunk 1]
```diff
@@ -4,17 +4,15 @@
...
-    src: "config.ini.j2"
-    dest: "{{ icingaweb2_modules_config_dir }}/icingadb/{{ _file }}.ini"
-    owner: "{{ icingaweb2_httpd_user }}"
-    group: "{{ icingaweb2_group }}"
-    mode: "0660"
-  loop:
-    - config
-    - resources
+- name: Create icingadb-web config files
+  include_tasks: manage_icingadbweb_config.yml
```
[the remark this answers is not in the document]
Comment (mocdaniel) [reply roles/icingaweb2/tasks/modules/icingadb_web.yml]: Done

**Review by mkayontour — APPROVED** [review]

**Comment by mocdaniel** [comment]
Do you want me to resolve the conflicts @mkayontour?

**Comment by mkayontour** [comment]
It's ok, I added some stuff and moved some variables. I'll clean up my stuff.

**Review by mkayontour — APPROVED** [review]

**Pull request merged**